### DISCUSSION TITLE
Check links automatically

### DISCUSSION THREAD

**Discussion by pattacini** [icub-tech-iit/documentation#94] [Ideas]
Hi @traversaro 

I've just seen that you starred https://github.com/linkchecker/linkchecker ⭐
Then, we might consider using this link-checking utility to crawl our documentation for broken links.

This can be done in the GH Action.
We need to properly set up the tool though.

**Comment by traversaro** [comment]
Yes, I set it up some time ago in https://github.com/robotology-playground/codyco-docs-linkchecker, but some recent link breakages (caused by the new IIT website) reminded that we should set it up it again (with a `schedule` job, to detect regressions in external links).

**Reply by pattacini** [reply]
Implemented in #139.

**Reply by pattacini** [reply]
Actually, `linkchecker` found 3 errors ✨
See https://github.com/icub-tech-iit/documentation/runs/2906462464?check_suite_focus=true.

**Reply by pattacini** [reply]
Fixed!

**Reply by maggia80** [reply]
cool!!!
